docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -363,53 +390,79 @@ Reboot the flight controller and connect to it again. This time in the "Messages
    
-  * For DSHOT, you cannot go straight to a high throttle level, as the motor will disarm after each command, and only re-arms on a throttle command close to 0%. So for DSHOT testing
-    increase the duration and send a low level throttle command to arm the motor, and before that command ends and the motor disarms, send another higher throttle command.  
+  * For DSHOT, you cannot go straight to a high throttle level, as the module will disarm after each command, and only re-arms on a throttle command close to 0%. So for DSHOT testing
+    increase the duration and send a low level throttle command to arm the module, and before that command ends and the module disarms, send another higher throttle command.  
 
-See the `Successful Test Videos`_ section below for a video of this whole process, which demonstrates what sounds you should expect from the motor and it successfully
+See the `Successful Test Videos`_ section below for a video of this whole process, which demonstrates what sounds you should expect from the module and it successfully
 spinning with Arducopter and Mission Planner.
 
 PX4 and QGroundControl Configuration and Testing
 ================================================
-This tutorial was tested using the stable version of QGroundControl from April 30, 2021 and PX4 versopm 1.12.3, as shown in the figures below. 
-The instructions assume you are starting from default parameters, and cover how to test that the flight controller and motor can communicate with no additional flight controller peripherals. See the `Hardware Setup`_ section for more details on the hardware and connections.
+This tutorial was tested using QGroundControl 4.2.4 and PX4 version 1.13.3, as shown in the figures below. 
+These instructions assume you are starting from default parameters, and cover how to test that the flight controller and module can communicate with no additional flight controller peripherals. 
+See the `Hardware Setup`_ section for more details on the hardware and connections.
 
-.. figure:: ../_static/tutorial_images/pwm_flight_controller/qgc_software.JPG
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/qgc_software.png
     :align: center
 
-    QGroundControl Sotfware Version
+    QGroundControl Sotfware Version Used for This Tutorial
 
-.. figure:: ../_static/tutorial_images/pwm_flight_controller/qgc_firmware.JPG
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/qgc_firmware.png
     :align: center
 
-    PX4 Firmware Version
+    PX4 Firmware Version Used for This Tutorial
 
 Setting Frame Type
 ******************
-The first step is to setup an airframe for the "vehicle". Normally this would be determined by the layout of your airframe, but since this is just a test with a single motor,
+The first step is to choose an airframe. Normally this would be determined by the layout of your airframe, but since this is just a test with a single module,
```

Comment:
Normally, this would be determined by the layout of your vehicle, but with only a single module, the exact airframe used isn't particularly important.